src/IssueViz.Security.UnitTests/Hotspots/LocalHotspotStoreTests.cs

```diff
@@ -336,6 +336,35 @@ public void Refresh_NewServerHotspots_ExistingHotspotsRematched()
             .BeEquivalentTo(new IssuesChangedEventArgs(issueVisualizations, issueVisualizations));
     }
 
+    [TestMethod]
+    public void GetAll_ReviewedServerHotspots_Filters()
+    {
+        var serverStoreMock = new Mock<IServerHotspotStore>();
+        var serverHotspot1 = CreateEmptyServerHotspot(status: "TO_REVIEW");
+        var serverHotspot2 = CreateEmptyServerHotspot(status: "REVIEWED", resolution: "ACKNOWLEDGED");
+        var serverHotspot3 = CreateEmptyServerHotspot(status: "REVIEWED", resolution: "FIXED"); //Expected to be filtered out
```

Comment:
It could also be marked as False Positive. Could be useful to add that to the test

Reply:
Or won't fix, can't remember

Reply:
The UI options look like this - not sure what the stored values are though:

![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/d787a1f3-4c7a-453d-a72f-54e640ffab94)

Reply:
[API docs](https://next.sonarqube.com/sonarqube/web_api/api/hotspots/search):

Status: REVIEWED or TO_REVIEW
Resolution: FIXED, SAFE, ACKNOWLEDGED.

So "SAFE" is the only resolution that isn't covered by tests.